### ISSUE TITLE
Questions about the `NegativeLogLikelihood`

### ISSUE THREAD

**Issue by huangmozhilv** [czifan/DeepSurv.pytorch#7]
I have two questions considering the computation of `NegativeLogLikelihood` in the [code](https://github.com/czifan/DeepSurv.pytorch/blob/master/networks.py#L74).
1. Why use `mask[(y.T - y) > 0] = 0`? In my view, this mask is supposed to indicate if Y_i>Y_j, where Y is survival time and i, j are two random observations, thus I think the code should be `mask[(y.T - y) <= 0] = 0`.
2. Why divide the log_loss in `log_loss = torch.sum(log_loss, dim=0) / torch.sum(mask, dim=0)`? According to the cox's negative log partial likelihood equation([link](https://mathweb.ucsd.edu/~rxu/math284/slect5.pdf)), there's no need to apply this division.

**Comment by czifan** [comment]
Thanks for your questions.
1. mask[(y.T - y) <= 0] = 0 is also available, requiring only 76 lines of sum's "dim" parameter to be modified. The two need to correspond strictly.
2. The original formula did not do this division. However, it will be an unstable training process when I throw away the division. This is why I apply this division.
